src/devices/Hcsr04/Hcsr04.cs

```diff
@@ -77,7 +79,7 @@ private double GetDistance()
             {
                 if (TryGetDistance(out double result))
```

Comment:
nit: I'd vote for TryGetDistance to return Length directly

Reply:
done